text/2021-02-26-chaos-engg-as-service.md

```diff
@@ -0,0 +1,79 @@
+# Chaos Engineering as a Service (CAAS)
+
+## Summary
+
+Build a unified dashboard to manage and monitor experiments for multiple
+platforms and multiple clusters.
+
+Related Issue: https://github.com/chaos-mesh/chaos-mesh/issues/1462
+
+## Motivation
+
+Following problem exists with the current chaos-mesh design:
+
+1. Cost of maintainance and change is high, since chaos-daemon and chaosd are
+   two seperate programs that serve similar purpose.
+   **Goal is to unify the two.**
```

Comment:
Chaosd is still immature and lacks many features,  we need to improve the tool. 
We had considered unifying chaos-daemon with Chaosd last year, however, this will increases the code complexity of the Chaos Mesh, so we will not unify these two into one.

Reply:
Yep fixed that